packages/esm-patient-registration-app/src/patient-registration/input/custom-input/autosuggest/autosuggest.test.tsx

```diff
@@ -79,7 +79,7 @@ describe('autosuggest', () => {
     fireEvent.change(searchbox, { target: { value: 'john' } });
     const listitems = await waitFor(() => screen.getAllByRole('listitem'));
     fireEvent.click(listitems[0]);
-    expect(handleSuggestionSelected).toHaveBeenNthCalledWith(1, 'person', 'randomuuid1');
+    expect(handleSuggestionSelected).toHaveBeenNthCalledWith(4, 'person', 'randomuuid1');
```

Comment:
Do we have an idea why this is now the 4th call rather than the first?

Reply:
Yes, because now every time you write something it clear the old value for that call the `handleSuggestionSelected` with an empty value.